File: app/src/com/ternaryop/photoshelf/adapter/OnPhotoBrowseClick.java
package com.ternaryop.photoshelf.adapter;

import android.view.View;

public interface OnPhotoBrowseClick {
	public void onPhotoBrowseClick(PhotoShelfPost post);
	public void onThumbnailImageClick(PhotoShelfPost post);
    public void onOverflowClick(View view, PhotoShelfPost post);
}

File: app/src/main/java/com/ternaryop/photoshelf/adapter/OnPhotoBrowseClick.kt
package com.ternaryop.photoshelf.adapter

import android.view.View

interface OnPhotoBrowseClick {
    fun onTagClick(position: Int, clickedTag: String)
    fun onThumbnailImageClick(position: Int)
    fun onOverflowClick(position: Int, view: View)
}
